config/jobs/kubernetes/sig-testing/files-remake.yaml

```diff
@@ -1,9 +1,47 @@
+periodics:
+- interval: 6h
+  kubernetes/kubernetes:
```

Comment:
checkconfig is complaining about this, you'll want to do something more like https://github.com/kubernetes/test-infra/blob/a4f5d39d933a5ce10cf101515322ec19d308c0fd/config/jobs/kubernetes/sig-testing/integration.yaml#L31-L40